requirements/dev.in

```diff
@@ -2,7 +2,7 @@
 
 black==23.1.0
 flake8==6.0.0
-isort==5.10.1
+isort==5.12.0
```

Comment:
Don't forget to bump the pre-commit as well :)

Reply:
pre-commit was already bumped, so looks like this one was missed (by me, prob) in the past